report_thesis/src/sections/problem_definition.tex

```diff
@@ -0,0 +1,54 @@
+\section{Problem Definition}
```

Comment:
Generally this section is really good. However, I am not sure whether or not the introduction to this section fits here. 
That said, it may. I think we should have a discussion about it and perhaps look into what other papers do.

My reason for questioning it is purely based on my understanding that this section should be almost purely "mathy" and as such a lot of the things explained in the introduction should be elsewhere.

Reply:
Following this logic, the same could perhaps be said for the ending as well. Maybe this is more of a problem statement? Let's discuss it.

Reply:
See my message in our chat. Not relevant here. But yes, let's talk abuot it.